alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/model/User.kt

```diff
@@ -0,0 +1,92 @@
+package com.amazon.opendistroforelasticsearch.alerting.model
+
+import org.elasticsearch.common.io.stream.StreamInput
+import org.elasticsearch.common.io.stream.StreamOutput
+import org.elasticsearch.common.io.stream.Writeable
+import org.elasticsearch.common.xcontent.ToXContent
+import org.elasticsearch.common.xcontent.ToXContentObject
+import org.elasticsearch.common.xcontent.XContentBuilder
+import org.elasticsearch.common.xcontent.XContentParser
+import org.elasticsearch.common.xcontent.XContentParserUtils.ensureExpectedToken
+import java.io.IOException
+
+data class User(
+    val name: String,
+    val backendRoles: List<String>,
+    val roles: List<String>,
+    val customAttNames: List<String>
+) : Writeable, ToXContentObject {
+
+    @Throws(IOException::class)
+    constructor(sin: StreamInput): this(
+            sin.readString(), // name
```

Comment:
nit: I prefer using the named arguments instead of comments
i.e.
```kotlin
@Throws(IOException::class)
constructor(sin: StreamInput): this(
    name = sin.readString(),
    backendRoles = sin.readStringList(),
    roles = sin.readStringList(),
    customAttNames = sin.readStringList() 
)
```

Reply:
+1. Still kotlin newbie